<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0014)about:internet -->
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <meta name="generator" content="Adobe RoboHelp 11" />
<title>Iso-Seq Application</title>
<link rel="StyleSheet" href="PacBio.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
-->
</style>
<script type="text/javascript" language="javascript1.2" src="whmsg.js" charset="utf-8"></script>
<script type="text/javascript" language="javascript" src="whver.js" charset="utf-8"></script>
<script type="text/javascript" language="javascript1.2" src="whutils.js" charset="utf-8"></script>
<script type="text/javascript" language="javascript1.2" src="whproxy.js" charset="utf-8"></script>
<script type="text/javascript" language="javascript1.2" src="whlang.js" charset="utf-8"></script>
<script type="text/javascript" language="javascript1.2" src="whtopic.js" charset="utf-8"></script>

</head>

<body><script type="text/javascript" language="javascript1.2">//<![CDATA[
<!--
if (window.gbWhTopic)
{
	var strUrl = document.location.href;
	var bc = 0;
	var n = strUrl.toLowerCase().indexOf("bc-");
	if(n != -1)
	{
		document.location.replace(strUrl.substring(0, n));
		bc = strUrl.substring(n+3);
	}

	if (window.addTocInfo)
	{
	addButton("show",BTN_TEXT,"Show","","","","",0,0,"","","");

	}
	if (window.writeBtnStyle)
		writeBtnStyle();

	if (window.writeIntopicBar)
		writeIntopicBar(1);

	
	document.write("<p style=\"text-align:right\"> ");
AddMasterBreadcrumbs("SL_Home.htm", "", "&gt;", "Home", "SL_Home_1.htm");
document.write("Iso-Seq Application<\/p>");

}
else
	if (window.gbIE4)
		document.location.reload();

onLoadHandler = function()
{
  if (window.setRelStartPage)
	{
	setTimeout("setRelStartPage('SL_Home.htm');", 1)

    setTimeout("UpdateBreadCrumbsMarker();", 1);
	}
}

if (window.addEventListener){  
	window.addEventListener('load', onLoadHandler, false);   
} else if (window.attachEvent){  
	window.attachEvent('onload', onLoadHandler);  
}

function onSetStartPage()
{
  autoSync(0);
  sendSyncInfo();
  sendAveInfoOut();
}
//-->
//]]></script>
<script type="text/javascript" src="./ehlpdhtm.js" language="JavaScript1.2"></script>

<div style="width: 100%; position: relative;" id="header">
	<h2 style="border-bottom-style: Solid; border-bottom-width: 1px; font-size: 14pt;"><span style="font-size: 14pt; "><font size="4" style="font-size:14pt;"><span 
	 style="font-family: Arial, sans-serif;">SMRT Analysis</span> Help</font></span></h2>
</div>
<h1>Iso-Seq Application </h1>
<p style="font-weight: normal;">Use this application to characterize transcripts 
 and splice variants. (Iso-Seq is a <span style="font-style: italic;"><i>de 
 novo</i></span> application.)</p>
<p style="font-weight: normal;">&#160;</p>
<p style="font-weight: normal;">The Iso-Seq application:</p>
<ul style="list-style: disc;">
	<li style="margin-left: -20px;"><p style="font-weight: normal;">Supports 
	 transcriptomics applications.</p></li>
	<li style="margin-left: -20px;"><p style="font-weight: normal;">Enables 
	 functional characterization of transcripts and splice variants analyzed 
	 from the PacBio platform.</p></li>
</ul>
<p style="font-weight: normal;">&#160;</p>
<p style="font-weight: normal;">This application generates full-length 
 cDNA sequences - from the 5â€™ end of transcripts to the poly-A tail - eliminating 
 the need for transcriptome reconstruction using isoform-inference algorithms. 
 </p>
<p style="font-weight: normal;">The Iso-Seq method generates accurate information 
 about alternatively spliced exons and transcriptional start sites. <span 
	 style="font-weight: normal;">It also delivers information about poly-adenylation 
 sites for transcripts up to 10 kb in length across the full complement 
 of isoforms within targeted genes or the entire transcriptome.</span></p>
<p style="font-weight: normal;">&#160;</p>
<p style="font-weight: normal;">The application includes three main steps:</p>
<ol style="list-style: decimal;">
	<li style="margin-left: -20px;"><p style="font-weight: normal;"><span 
	 style="font-weight: bold;">CCS: </span>Build circular consensus sequences 
	 (CCSs) from the subreads. </p></li>
	<li style="margin-left: -20px;"><p style="font-weight: normal;"><span 
	 style="font-weight: bold;">Classify</span>: Identify and remove polyA/T 
	 tails, remove primers, and identify read-strandedness. Also removes 
	 artificial concatemers, but does <span style="font-weight: bold;">not</span> 
	 remove PCR chimeras.</p></li>
	<li style="margin-left: -20px;"><p style="font-weight: normal;"><span 
	 style="font-weight: bold;">Cluster</span>: Performs <span style="font-style: italic;"><i>de 
	 novo</i></span> clustering and consensus calling. Outputs polished, full-length, 
	 high-quality consensus isoforms that are further separated into high-quality 
	 (HQ) and low-quality (LQ) based on predicted accuracies.</p></li>
</ol>
<p style="font-weight: normal;">&#160;</p>
<table class="PacBioColor" cellspacing="0px" width="59.657%">
	<col style="width: 30.527%;" />
	<col style="width: 8.504%;" />
	<col style="width: 60.969%;" />
	<tr class="t1st">
		<td style="vertical-align: Top;"><p style="text-align: center; 
											 font-family: Arial, sans-serif; 
											 font-size: 10pt;">Advanced 
		 Analysis Parameters - CCS</p></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">Default 
		 Value</p></td>
		<td style="vertical-align: Top;"><p style="text-align: center; 
											 font-family: Arial, sans-serif; 
											 font-size: 10pt;">Description</p></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-weight: bold; 
												 font-size: 10pt; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Minimum 
		 Predicted Accuracy</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">0.8</p></td>
		<td style="vertical-align: Top;"><span style="font-size: 10pt; 
												 font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">The 
		 </font></span><span style="font-size: 10pt; font-weight: bold; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">minimum</font></span><span 
		 style="font-size: 10pt; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;"> predicted 
		 accuracy of a read, ranging from 0 to 1. (0.99 indicates that 
		 only reads expected to be 99% accurate are emitted.)</font></span></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-weight: bold; 
												 font-size: 10pt; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Minimum 
		 Number of Passes</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">1</p></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif;">The 
		 minimum number of <span style="font-weight: bold;">full</span> 
		 passes for a ZMW to be emitted. Full passes <span style="font-weight: bold;">must</span> 
		 have an adapter hit before and after the insert sequence and so 
		 does not include any partial passes at the start and end of the 
		 sequencing reaction.</p></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-weight: bold; 
												 font-size: 10pt; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Minimum 
		 Z Score</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">-9999</p></td>
		<td style="vertical-align: Top;"><span style="font-size: 10pt; 
												 font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">The 
		 minimum Z-Score for a subread to be included in the consensus 
		 generating process.</font></span></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><p style="font-weight: bold; font-family: Arial, sans-serif;">Maximum 
		 Subread Length</p></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">15,000</p></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif;">The 
		 maximum length for the median size of insert reads to generate 
		 a consensus sequence. If the targeted template is known to be 
		 a particular size range, this can filter out alternative DNA templates.</p></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><p style="font-weight: bold; font-family: Arial, sans-serif;">By 
		 Strand CCS</p></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">OFF</p></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif;">For 
		 each ZMW, generate two CCS sequences - one for each strand.</p></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><p style="font-weight: bold; font-family: Arial, sans-serif;">Minimum 
		 Subread Length</p></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">300</p></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif;">The 
		 minimum length for the median size of insert reads to generate 
		 a consensus sequence. If the targeted template is known to be 
		 a particular size range, this can filter out alternative DNA templates.</p></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-weight: bold; 
												 font-size: 10pt; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Minimum 
		 Read Score</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">0.65</p></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif;">The 
		 minimum read score of input subreads.</p></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-family: Arial, sans-serif; 
												 font-size: 10pt; font-weight: bold;"><font size="2" style="font-size:10pt;">Maximum 
		 Dropped Fraction</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">0.8</p></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif;">The 
		 maximum fraction of subreads that can be dropped before giving 
		 up.</p></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-weight: bold; 
												 font-size: 10pt; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Minimum 
		 SNR</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">3.75</p></td>
		<td style="vertical-align: Top;"><span style="font-size: 10pt; 
												 font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">The 
		 minimum required signal-to-noise ratio (SNR) for any of the four 
		 channels. Data with SNR &lt; 3.75 is typically considered lower 
		 quality.</font></span></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-weight: bold; 
												 font-size: 10pt; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">No 
		 Polish CCS</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">ON</p></td>
		<td style="vertical-align: Top;"><span style="font-size: 10pt; 
												 font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Specify 
		 whether or not to polish CCS reads using Arrow.</font></span></td>
	</tr>
</table>
<p style="font-weight: normal;">&#160;</p>
<table class="PacBioColor" cellspacing="0px" width="59.592%">
	<col style="width: 30.561%;" />
	<col style="width: 8.513%;" />
	<col style="width: 60.926%;" />
	<tr class="t1st">
		<td style="vertical-align: Top;"><p style="text-align: center; 
											 font-family: Arial, sans-serif; 
											 font-size: 10pt;">Advanced 
		 Analysis Parameters - IsoSeq</p></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">Default 
		 Value</p></td>
		<td style="vertical-align: Top;"><p style="text-align: center; 
											 font-family: Arial, sans-serif; 
											 font-size: 10pt;">Description</p></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-family: Arial, sans-serif; 
												 font-size: 10pt; font-weight: bold;"><font size="2" style="font-size:10pt;">Bin 
		 by Read Length Manually</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;"><span 
		 style="font-family: Arial, sans-serif;">NONE</span></p>
		<p>&#160;</p></td>
		<td style="vertical-align: Top;"><span style="font-family: Arial, sans-serif; 
												 font-size: 10pt;"><font size="2" style="font-size:10pt;">Specify 
		 binning reads by manually inputting read length bins. (Example: 
		 [0, 2, 3, 5] means binning reads into 0-2KB, 2-3KB, 3-5KB, and 
		 above 5KB bins.) This overwrites the &quot;Bin By Read Length 
		 in KB&quot; option.</font></span></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-family: Arial, sans-serif; 
												 font-size: 10pt; font-weight: bold;"><font size="2" style="font-size:10pt;">Bin 
		 by Primer</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;"><span 
		 style="font-family: Arial, sans-serif;">OFF</span></p></td>
		<td style="vertical-align: Top;"><span style="font-family: Arial, sans-serif; 
												 font-size: 10pt;"><font size="2" style="font-size:10pt;">Specify 
		 binning reads by Primers. This overwrites the &quot;Bin by Read 
		 Length Manually&quot; and &quot;Bin by Read Length in KB&quot; 
		 options.</font></span></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-family: Arial, sans-serif; 
												 font-size: 10pt; font-weight: bold;"><font size="2" style="font-size:10pt;">Minimum 
		 Accuracy of Polished Isoforms</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">0.99</p></td>
		<td style="vertical-align: Top;"><span style="font-size: 10pt; 
												 font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">The 
		 </font></span><span style="font-size: 10pt; font-weight: bold; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">minimum</font></span><span 
		 style="font-size: 10pt; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;"> accuracy 
		 needed to classify an isoform as high-quality.</font></span></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-weight: bold; 
												 font-size: 10pt; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Bin 
		 by Read Length in KB</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">1</p></td>
		<td style="vertical-align: Top;"><span style="font-family: Arial, sans-serif; 
												 font-size: 10pt;"><font size="2" style="font-size:10pt;">Specify 
		 binning reads by the read's read length in KB.</font></span></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-size: 10pt; 
												 font-weight: bold; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Trim 
		 QVs 3'</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">30</p></td>
		<td style="vertical-align: Top;"><span style="font-size: 10pt; 
												 font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Specify 
		 the number of bases whose Quality Value to ignore in the 3' end.</font></span><span 
		 style="font-family: Arial, sans-serif;"> </span></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-family: Arial, sans-serif; 
												 font-size: 10pt; font-weight: bold;"><font size="2" style="font-size:10pt;">Sample 
		 Name</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">NONE</p></td>
		<td style="vertical-align: Top;"><span style="font-family: Arial, sans-serif; 
												 font-size: 10pt;"><font size="2" style="font-size:10pt;">The 
		 name of the input sample. A random string is used when Sample 
		 Name is NONE.</font></span></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-size: 10pt; 
												 font-weight: bold; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Ignore 
		 PolyA</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">OFF</p></td>
		<td style="vertical-align: Top;"><span style="font-size: 10pt; 
												 font-family: Arial, sans-serif; 
												 font-weight: bold;"><font size="2" style="font-size:10pt;">OFF</font></span><span 
		 style="font-size: 10pt; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;"> means 
		 that polyA tails are required for a sequence to be considered 
		 full length. </font></span><span style="font-size: 10pt; font-family: Arial, sans-serif; 
									 font-weight: bold;"><font size="2" style="font-size:10pt;">ON</font></span><span 
		 style="font-size: 10pt; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;"> means 
		 sequences do not need polyA tails to be considered full length. 
		 </font></span></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-size: 10pt; 
												 font-weight: bold; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Trim 
		 QVs 5'</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">100</p></td>
		<td style="vertical-align: Top;"><span style="font-size: 10pt; 
												 font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Specify 
		 the number of bases whose Quality Value to ignore in the 5' end.</font></span><span 
		 style="font-family: Arial, sans-serif;"> </span></td>
	</tr>
	<tr class="t1Row">
		<td style="vertical-align: Top;"><span style="font-size: 10pt; 
												 font-weight: bold; font-family: Arial, sans-serif;"><font size="2" style="font-size:10pt;">Minimum 
		 Sequence Length</font></span></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif; 
											 text-align: center;">300</p></td>
		<td style="vertical-align: Top;"><p style="font-family: Arial, sans-serif;">The 
		 minimum sequence length to output.</p></td>
	</tr>
</table>
<p>&#160;</p>
<div style="width: 100%; position: relative;" id="footer">
	<p style="font-size: 10pt; border-top-style: Solid; border-top-width: 1px;"><span style="font-size: 10pt; "><font size="2" style="font-size:10pt;"><a 
	 href="SL_Legal.htm">Legal Information</a> &#160;&#160;&#160;&#160;&#160;12-May-16</font></span></p>
</div>
<script type="text/javascript" language="javascript1.2">//<![CDATA[
<!--
if (window.writeIntopicBar)
	writeIntopicBar(0);


highlightSearch();
//-->
//]]></script>
</body>
</html>
