<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0014)about:internet -->
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <meta name="generator" content="Adobe RoboHelp 11" />
<title>Pacific Biosciences Terminology</title>
<link rel="StyleSheet" href="PacBio.css" type="text/css" />
<script type="text/javascript" language="JavaScript">
//<![CDATA[
function reDo() {
  if (innerWidth != origWidth || innerHeight != origHeight)
     location.reload();
}
if ((parseInt(navigator.appVersion) == 4) && (navigator.appName == "Netscape")) {
	origWidth = innerWidth;
	origHeight = innerHeight;
	onresize = reDo;
}
onerror = null; 
//]]>
</script>
<style type="text/css">
<!--
div.WebHelpPopupMenu { position:absolute;
left:0px;
top:0px;
z-index:4;
visibility:hidden; }
-->
</style>
<script type="text/javascript" language="javascript1.2" src="whmsg.js" charset="utf-8"></script>
<script type="text/javascript" language="javascript" src="whver.js" charset="utf-8"></script>
<script type="text/javascript" language="javascript1.2" src="whutils.js" charset="utf-8"></script>
<script type="text/javascript" language="javascript1.2" src="whproxy.js" charset="utf-8"></script>
<script type="text/javascript" language="javascript1.2" src="whlang.js" charset="utf-8"></script>
<script type="text/javascript" language="javascript1.2" src="whtopic.js" charset="utf-8"></script>

</head>

<body><script type="text/javascript" language="javascript1.2">//<![CDATA[
<!--
if (window.gbWhTopic)
{
	var strUrl = document.location.href;
	var bc = 0;
	var n = strUrl.toLowerCase().indexOf("bc-");
	if(n != -1)
	{
		document.location.replace(strUrl.substring(0, n));
		bc = strUrl.substring(n+3);
	}

	if (window.addTocInfo)
	{
	addTocInfo("SMRT Link\nPacBioTerminology");
addButton("show",BTN_TEXT,"Show","","","","",0,0,"","","");

	}
	if (window.writeBtnStyle)
		writeBtnStyle();

	if (window.writeIntopicBar)
		writeIntopicBar(1);

	
	document.write("<p style=\"text-align:right\"> ");
AddMasterBreadcrumbs("SL_Home.htm", "", "&gt;", "Home", "SL_Home_1.htm");
document.write("<a href=\"SL_CS_RunDes_Top.htm\">SMRT Link<\/a> &gt; PacBioTerminology<\/p>");

}
else
	if (window.gbIE4)
		document.location.reload();

onLoadHandler = function()
{
  if (window.setRelStartPage)
	{
	setTimeout("setRelStartPage('SL_Home.htm');", 1)

    setTimeout("UpdateBreadCrumbsMarker();", 1);
	}
}

if (window.addEventListener){  
	window.addEventListener('load', onLoadHandler, false);   
} else if (window.attachEvent){  
	window.attachEvent('onload', onLoadHandler);  
}

function onSetStartPage()
{
  autoSync(0);
  sendSyncInfo();
  sendAveInfoOut();
}
//-->
//]]></script>
<script type="text/javascript" src="./ehlpdhtm.js" language="JavaScript1.2"></script>

<div style="width: 100%; position: relative;" id="header">
	<h2 style="border-bottom-style: Solid; border-bottom-width: 1px; font-size: 14pt;"><span style="font-size: 14pt; "><font size="4" style="font-size:14pt;"><span style="font-family: Arial, sans-serif;">SMRT</span><span style="font-family: Arial, sans-serif; 
	 font-size: 10pt; vertical-align: Super;"><font size="2" style="font-size:10pt;">®</font></span><span style="font-family: Arial, sans-serif;"> 
 Link</span> 
	 Help</font></span></h2>
</div>
<h1>Pacific Biosciences Terminology</h1>
<h2>General Terminology</h2>
<ul style="list-style: disc;">
	<li style="margin-left: -20px;"><p><span style="font-weight: bold;"><span style="font-family: Arial, sans-serif;">SMRT</span><span style="font-family: Arial, sans-serif; 
	 font-size: 10pt; vertical-align: Super;"><font size="2" style="font-size:10pt;">®</font></span><span style="font-family: Arial, sans-serif;"> 
 Cell</span></span>: 
	 Consumable substrates comprising arrays of zero-mode waveguide nanostructures. 
	 SMRT Cells are used in conjunction with the <span style="font-weight: bold;">DNA 
	 Sequencing Kit</span> for on-instrument DNA sequencing.</p></li>
	<li style="margin-left: -20px;"><span style="font-weight: bold;">collection</span>: 
	 The set of data collected during real-time observation of the <span style="font-family: Arial, sans-serif;">SMRT</span><span style="font-family: Arial, sans-serif; 
	 font-size: 10pt; vertical-align: Super;"><font size="2" style="font-size:10pt;">®</font></span><span style="font-family: Arial, sans-serif;"> 
 Cell</span>; 
	 including spectral information and temporal information used to determine 
	 a read.</li>
	<li style="margin-left: -20px;"><span style="font-weight: bold;">zero-mode 
	 waveguide (ZMW)</span>: A nanophotonic device for confining light 
	 to a small observation volume. This can be, for example, a small hole 
	 in a conductive layer whose diameter is too small to permit the propagation 
	 of light in the wavelength range used for detection. Physically part 
	 of a <span style="font-family: Arial, sans-serif;">SMRT</span><span style="font-family: Arial, sans-serif; 
	 font-size: 10pt; vertical-align: Super;"><font size="2" style="font-size:10pt;">®</font></span><span style="font-family: Arial, sans-serif;"> 
 Cell</span>.</li>
	<li style="margin-left: -20px;"><span style="font-weight: bold;">Run</span>: 
	 Specifies</li>
	<li><p style="margin-left: 0px; font-size: 10pt;">The <span style="font-weight: bold;">wells</span> 
	 and <span style="font-family: Arial, sans-serif; font-weight: bold;">SMRT 
	 Cell</span><span style="font-weight: bold;">s</span> to include in 
	 the sequencing run.</p></li>
	<li><p style="margin-left: 0px; font-size: 10pt;">The collection protocols 
	 and analysis <span style="font-weight: bold;">applications </span>to 
	 use for the selected wells and cells.</p></li>
</ul>
<h2>Read Terminology</h2>
<ul style="list-style: disc;">
	<li style="margin-left: -20px; font-family: Arial, sans-serif; font-size: 10pt; 
		 color: #000000;"><span style="font-weight: bold;">polymerase read</span>: 
	 A sequence of nucleotides incorporated by the DNA polymerase while 
	 reading a template, such as a circular SMRTbell template. They can 
	 include sequences from adapters and from one or multiple passes around 
	 a circular template, which includes the insert of interest. Polymerase 
	 reads are most useful for quality control of the instrument run. Polymerase 
	 read metrics primarily reflect movie length and other run parameters 
	 rather than insert size distribution. Polymerase reads are trimmed 
	 to include only the high-quality region. <span style="font-weight: bold;">Note</span>: 
	 Sample quality is a major factor in polymerase read metrics.</li>
	<li style="margin-left: -20px; font-family: Arial, sans-serif; font-size: 10pt; 
		 color: #000000; font-weight: normal;"><span style="font-weight: bold;">subreads</span><span>: 
	 </span>Each polymerase read is partitioned to form one or more subreads, 
	 which contain sequence from a single pass of a polymerase on a single 
	 strand of an insert within a SMRTbell template and <span style="font-weight: bold;">no</span> 
	 adapter sequences. The subreads contain the full set of quality values 
	 and kinetic measurements. Subreads are useful for applications such 
	 as <span style="font-style: italic;"><i>de novo</i></span> assembly, resequencing, 
	 base modification analysis, and so on.</li>
	<li style="margin-left: -20px;"><p><span style="font-weight: bold;">circular 
	 consensus (CCS) read</span>: The consensus sequence resulting from 
	 alignment between subreads taken from a single ZMW. Note that generation 
	 of the CCS read does not include or require alignment against a reference 
	 sequence. Generation of CCS reads using the CCS algorithm requires 
	 at least two full-pass subreads from the insert.</p></li>
</ul>
<p><img src="3Part_Reads_Def.png" alt="" style="border: none;" width="629" 
		 height="262" border="0" /></p>
<h2>Read Length Terminology</h2>
<ul style="list-style: disc;">
	<li style="margin-left: -20px;"><p><span style="font-weight: bold;">mapped 
	 polymerase read length</span>: Approximates the sequence produced 
	 by a polymerase in a ZMW. The total number of bases along a read from 
	 the first adapter of aligned subread to the last adapter or aligned 
	 subread.</p></li>
	<li style="margin-left: -20px;"><p><span style="font-weight: bold;">mapped 
	 subread length</span>: The length of the subread alignment to a target 
	 reference sequence. This does <span style="font-weight: bold;">not</span> 
	 include the adapter sequence.</p></li>
</ul>
<p><img src="Mapped_Length.png" alt="" style="border: none;" width="522" 
		 height="181" border="0" /></p>
<ul style="list-style: disc;">
	<li style="margin-left: -20px;"><p><span style="font-weight: bold;">polymerase 
	 read length</span>: The total number of bases produced from a ZMW 
	 after trimming. This <span style="font-weight: bold;">may</span> include 
	 the adapter sequences.</p></li>
</ul>
<h2>Secondary Analysis Terminology</h2>
<ul style="list-style: disc;">
	<li style="margin-left: -20px;"><span style="font-weight: bold;">Secondary 
	 analysis: </span>Follows primary analysis and uses basecalled data. 
	 It is application-specific, and may include:</li>
	<li>Filtering/selection of data that meets a desired criteria (such 
	 as quality, read length, and so on).</li>
	<li><p>Comparison of reads to a reference or between each other for 
	 mapping and variant calling, consensus sequence determination, alignment 
	 and assembly <span style="font-style: italic;"><i>(de novo</i></span> or 
	 reference-based), variant identification, and so on.</p></li>
	<li><p>Quality evaluations for a sequencing run, consensus sequence, 
	 assembly, and so on.</p></li>
	<li><p>PacBio’s SMRT Analysis contains a variety of secondary analysis 
	 applications including RNA and Epigenomics analysis tools.</p></li>
	<li style="margin-left: -20px;"><p><span style="font-weight: bold;">secondary 
	 analysis application</span> (Formerly “Secondary analysis protocol”): 
	 A secondary analysis workflow that may include multiple analysis steps. 
	 Examples include <span style="font-style: italic;"><i>de novo</i></span> 
	 assembly, resequencing, RNA and epigenomics analysis.</p></li>
	<li style="margin-left: -20px;"><span style="font-weight: bold;">Consensus</span>: 
	 Generation of a consensus sequence from multiple-sequence alignment.</li>
	<li style="margin-left: -20px;"><span style="font-weight: bold;">Filtering</span>: 
	 Removes reads that do not meet the Read Quality and Read Length parameters 
	 set by the user. </li>
	<li style="margin-left: -20px;"><span style="font-weight: bold;">Mapping</span>: 
	 Local alignment of a read or subread to a reference sequence.</li>
</ul>
<h2>Accuracy Terminology</h2>
<ul style="list-style: disc;">
	<li style="margin-left: -20px;"><p><span style="font-weight: bold;">circular 
	 consensus accuracy</span>: Accuracy based on consensus sequence from 
	 multiple sequencing passes around a single circular template molecule.</p></li>
	<li style="margin-left: -20px;"><p><span style="font-weight: bold;">consensus 
	 accuracy</span>: Accuracy based on aligning multiple sequencing reads 
	 or subreads together.</p></li>
	<li style="margin-left: -20px;"><span style="font-weight: bold;">polymerase 
	 read quality</span>: A trained prediction of a read’s mapped accuracy 
	 based on its pulse and base file characteristics (peak signal-to-noise 
	 ratio, average base QV, inter-pulse distance, and so on).</li>
</ul>
<p>&#160;</p>
<div style="width: 100%; position: relative;" id="footer">
	<p style="font-size: 10pt; border-top-style: Solid; border-top-width: 1px;"><span style="font-size: 10pt; "><font size="2" style="font-size:10pt;"><a 
	 href="SL_Legal.htm">Legal Information</a> &#160;&#160;&#160;&#160;&#160;12-May-16</font></span></p>
</div>
<script type="text/javascript" language="javascript1.2">//<![CDATA[
<!--
if (window.writeIntopicBar)
	writeIntopicBar(0);


highlightSearch();
//-->
//]]></script>
</body>
</html>
